pages/success.tsx

```diff
@@ -1,10 +1,31 @@
-import { Card, Heading, SmallText } from "@cheapreats/react-ui";
+import { Card, Heading, SmallText, Button } from "@cheapreats/react-ui";
 import React from "react";
 import styled from "styled-components";
 import Image from "next/image";
 import Snowfall from "react-snowfall";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
 
 export const Success: React.VFC = () => {
+  const [orderId, setOrderId] = useState("");
+  const router = useRouter();
+
+  useEffect(() => {
+    if (!!router.query.id) {
+      setOrderId(router.query.id as string);
+    }
+  }, [router.query]);
+
+  function newOrder() {
+    window.location.replace("/order");
+  }
+
+  useEffect(() => {
+    const duration = 7000;
```

Comment:
Const at top of the file, no need to redeclare.
Duration is a bad variable name.
What is the meaning! The meaning is
SESSION_EXPIRY_TIMEOUT = 7000;
Much more intitutife when you read the variable versus just 7000